src/main/resources/TotemCode/SimpleTotemItemSheet.xml

```diff
@@ -12,8 +12,8 @@
   <customClass/>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
   <creationDate>1398855257000</creationDate>
-  <date>1422027885000</date>
```

Comment:
We should avoid date changes. The commit should normally use the same date as before. This is not a blocker so it can be commited as is but it would be better to avoid this next time.